httpobs/database/database.py

```diff
@@ -14,7 +14,7 @@
                           SCANNER_ABORT_SCAN_TIME)
 from httpobs.scanner import STATE_ABORTED, STATE_FAILED, STATE_FINISHED, STATE_PENDING, STATE_STARTING
 from httpobs.scanner.analyzer import NUM_TESTS
-from httpobs.scanner.grader import get_grade_for_score
+from httpobs.scanner.grader import get_grade_and_likelihood_indicator_for_score
```

Comment:
likelihood_indicator is less problematic as a column name; I care a lot less about it there.  :)